oap-server/server-bootstrap/src/main/java/org/apache/skywalking/oap/server/starter/config/ApplicationConfigLoader.java

```diff
@@ -99,11 +103,26 @@ private void loadConfig(ApplicationConfiguration configuration) throws ConfigFil
     }
 
     private void replacePropertyAndLog(final Object propertyName, final Object propertyValue, final Properties target,
-        final Object providerName) {
-        final Object replaceValue = yaml.load(PropertyPlaceholderHelper.INSTANCE.replacePlaceholders(propertyValue + "", target));
-        if (replaceValue != null) {
-            target.replace(propertyName, replaceValue);
-            logger.info("The property with key: {}, value: {}, in {} provider", propertyName, replaceValue.toString(), providerName);
+                                       final Object providerName) {
+        final String valueString = PropertyPlaceholderHelper.INSTANCE
+            .replacePlaceholders(propertyValue + "", target);
+        if (valueString != null) {
```

Comment:
`if (valueString != null) {` is always true

Reply:
This is the existing logic, I didn't change it.